mithril-stm/src/stm.rs

```diff
@@ -653,12 +423,13 @@ impl<D: Digest + Clone + FixedOutput> StmClerk<D> {
     /// Create a Clerk from a signer.
     pub fn from_signer(signer: &StmSigner<D>) -> Self {
         let closed_reg = signer
-            .closed_reg
+            .get_closed_reg()
             .clone()
-            .expect("Core signer does not include closed registration. StmClerk, and so, the Stm certificate cannot be built without closed registration!");
+            // .expect("Core signer does not include closed registration. StmClerk, and so, the Stm certificate cannot be built without closed registration!")
+            ;
```

Comment:
I think that this `expect` should be kept (to be iso with the previous development).

In the long run , we will need to replace it with proper error handling and the function would be replaced with `pub fn try_from_signer(signer: &StmSigner<D>) -> StdResult<Self>`.